lib/models/index.js

```diff
@@ -13,10 +13,11 @@ var dbconfig = config.db;
 dbconfig.logging = config.debug ? logger.info : false;
 
 var sequelize = null;
-if (dbconfig.hasOwnProperty('username') || dbconfig.hasOwnProperty('password'))
-    sequelize = new Sequelize(dbconfig.database, dbconfig.username, dbconfig.password, dbconfig);
+
+if (process.env.DATABASE_URL)
+    sequelize = new Sequelize(process.env.DATABASE_URL, dbconfig.username, dbconfig.password, dbconfig);
```

Comment:
For heruko this is may better but you don't pass useful parameters to object constructor like `pool` configs etc. The last parameter is very powerful and shouldn't removed even if it's possible. On one hand for backwards compatibility reasons (may someone uses it) on the other hand to may optimize database handling later.